x-pack/plugins/security_solution/public/helpers.ts

```diff
@@ -143,7 +143,7 @@ export const getInspectResponse = <T extends FactoryQueryTypes>(
 
 export const isDetectionsPath = (pathname: string): boolean => {
   return !!matchPath(pathname, {
-    path: `(${ALERTS_PATH}|${RULES_PATH}|${EXCEPTIONS_PATH})`,
+    path: `(${ALERTS_PATH}|${RULES_PATH}|${UEBA_PATH}|${EXCEPTIONS_PATH})`,
```

Comment:
ueba will be powered by alerts, so we need the default siem alert index in the sourcerer